src/jitlayers.cpp

```diff
@@ -1865,6 +1909,48 @@ Error JuliaOJIT::addObjectFile(orc::JITDylib &JD, std::unique_ptr<MemoryBuffer>
     return LockLayer.add(JD.getDefaultResourceTracker(), std::move(Obj));
 }
 
+#if JL_LLVM_VERSION >= 170000
+Expected<ExecutorSymbolDef> JuliaOJIT::findSymbol(StringRef Name, bool ExportedSymbolsOnly)
```

Comment:
Did only the return type change?

Reply:
Pretty much, ExecutorSymbolDef is the new `JITEvaluatedSymbol`. It has a bit more type safety and supports authentication but the API is almost one to one wiht the exception of taking raw pointers which requires a `fromPtr` call